pages/common/fortune.md

```diff
@@ -0,0 +1,27 @@
+# fortune
+
+> Print a random quotation (fortune-cookie style)
+
+- Print a quotation
+
+`fortune`
+
+- Print a quotation from a given database
+
+`fortune {{database}}`
+
+- Print a list of quotation databases
```

Comment:
I would avoid using word "file". File is overused term. Actually everything is file in Unix.

If a newbie see such an explanation, the first thing he can do:
`fortune /path/to/my/stupid/README.md`
And then he will be surprised that it does not work.

I like database, because it is database in fact, it is just not relational database. It is what is described here:
https://en.wikipedia.org/wiki/Database?oldformat=true#Terminology_and_overview

I would agree to word "dictionary" too. Or vocabulary. Maybe I should add once that it is a file of certain format.